docroot/themes/humsci/su_humsci_gin_admin/su_humsci_gin_admin.theme

```diff
@@ -0,0 +1,57 @@
+<?php
+
+/**
+ * @file
+ * Theme functions for su_humsci_gin_admin theme.
+ */
+
+use Drupal\Component\Utility\Html;
+use Drupal\Core\Template\Attribute;
+use Drupal\paragraphs\ParagraphInterface;
+
+/**
+ * Implements hook_library_info_alter().
+ */
+function su_humsci_gin_admin_library_info_alter(&$libraries, $extension) {
+  $libraries_unset = [
+    'humsci_traditional' => 'humsci-colorful-styling',
+    'humsci_colorful' => 'humsci-traditional-styling',
+  ];
+  if ($extension === 'su_humsci_gin_admin') {
+    $theme = \Drupal::config('system.theme')->get('default');
+    unset($libraries[$libraries_unset[$theme]]);
+  }
+}
+
+/**
+ * Implements hook_preprocess_HOOK().
+ */
+function su_humsci_gin_admin_preprocess(&$variables, $hook) {
+  if (!(str_starts_with($hook, 'pattern_') || $hook == 'paragraph')) {
+    return NULL;
+  }
+  $theme = \Drupal::config('system.theme')->get('default');
+  $color_pairing = theme_get_setting('theme_color_pairing', $theme);
+  $prefix = [
+    'humsci_traditional' => 'ht',
+    'humsci_colorful' => 'hc',
+  ];
+  $variables['wrapper_attributes'] = new Attribute([
+    'class' => [
+      $prefix[$theme] . '-pairing-' . $color_pairing,
+    ],
+  ]);
+  if ($hook === 'paragraph') {
+    $variables['wrapper_attributes']->addClass('paragraph-item', 'ptype-' . Html::cleanCssIdentifier($variables['paragraph']->getType()));
```

Comment:
It might be cleaner to create a $classes array and add the classes to it, then at the end of code create the attribute object and add all the classes to it at once.

```suggestion
  $classes = [];
  $classes[] = $prefix[$theme] . '-pairing-' . $color_pairing;

  if ($hook === 'paragraph') {
    $classes[] = 'paragraph-item';
    $classes[] = 'ptype-' . Html::cleanCssIdentifier($variables['paragraph']->getType());
```

Continue to add the classes to `$classes` then at the bottom:

```
  $variables['wrapper_attributes'] = new Attribute();
  $variables['wrapper_attributes']->addClass($classes);
```

Reply:
Fixed!